### PR TITLE
Changes to README

### PR DESCRIPTION
**Pull request by vanguard07**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,15 +17,15 @@
...
 | N | Play next track |
 | R | Replay current track |
 | F | Toggle fullscreen |
 | L | Play last played track / Track on the end of the list |
-| UP | Volume Up |
-| DOWN | Volume Down |
-| LEFT | Track Seek -10s |
-| RIGHT | Track Seek +10s |
+| UP | Volume up in 10% increments |
+| DOWN | Volume down in 10% increments |
```
Comment (anshumanv) [inline README.md]: Volume down in 10% increments ? makes sense

Comment (vanguard07) [reply README.md]: Means volume goes up (eg: from 10 to 20) from the up button. Same for the down button.

Comment (aashutoshrathi) [reply README.md]: Better if we make it
```md
| UP | Increment in volume by 10% |
| DOWN | Decrement in volume by 10% |
```

Comment (anshumanv) [reply README.md]: ^ Perfect

Comment (anshumanv) [reply README.md]: That 'in' from both lines is also not needed maybe ?

Comment (vanguard07) [reply README.md]: @aashutoshrathi: Better than mine! Great! 👍

Comment (aashutoshrathi) [reply README.md]: @Anshuman-Verma Do you mean?

Better if we make it
```md
| UP | Increments volume by 10% |
| DOWN | Decrements volume by 10% |
```

Comment (vanguard07) [reply README.md]: Make it "Increments/Decrements"

Comment (aashutoshrathi) [reply README.md]: GFI @vanguard07

**Comment by vanguard07** [comment]
Updated!!!

**Comment by aashutoshrathi** [comment]
Can we make something better instead of `jump` like `seek`

**Comment by vanguard07** [comment]
Jump seems better than seek. More understandable.

**Review by aashutoshrathi — APPROVED** [review]
👍

**Comment by aashutoshrathi** [comment]
@Anshuman-Verma Please check this one and Merge if things are in place.

**Review by anshumanv — APPROVED** [review]
Looks well

**Pull request merged**